app/controllers/job_runs_controller.rb

```diff
@@ -42,16 +43,6 @@ def download_report
     end
   end
 
-  def discovery_report_summary
-    job_run = JobRun.find(params[:id])
-    if job_run.output_location && File.exist?(job_run.output_location)
-      @discovery_report = JSON.parse(File.read(job_run.output_location))
-    else
-      flash.now[:warning] = 'Discovery report file is not available.'
-      render 'show'
-    end
-  end
-
   private
```

Comment:
this is now just showed inline in the page without a separate link...hopefully we won't run into performance issues (though I think a very large discovery report file is not common, since the summary section is always the same size regardless of number of objects, so that would mean a lot of individual objects with errors)